Makefile

```diff
@@ -0,0 +1,4 @@
+GOALS := $(or $(MAKECMDGOALS),all)
```

Comment:
I think this is a good intermediate step until we move the `Makefile` to the root in a future PR